fees/catex/index.ts

```diff
@@ -0,0 +1,50 @@
+import { FetchOptions, SimpleAdapter } from '../../adapters/types';
+import { CHAIN } from '../../helpers/chains';
+import { getConfig } from '../../helpers/cache';
+import { EventFragment, id, zeroPadValue } from 'ethers';
+
+const STRATEGIES_URL = 'https://raw.githubusercontent.com/Lynexfi/lynex-lists/main/strategies/main.json';
+const POOLMANAGER = '0x67366782805870060151383f4bbff9dab53e5cd6'; // Uniswap v4 PoolManager on Polygon
+const COLLECT_EVENT = 'event Collect(bytes32 indexed poolId, address indexed recipient, uint256 amount0, uint256 amount1)';
```

Comment:
are you sure that this event is correct? I am not see any logs with topic0 matching this hash

Reply:
topic hash = `0x0dffebc0f562cda9f14ea495ab21fd21b4791ae0b16da1a93010a86f0729f4fd`

Reply:
Thank u for pointing out. I've now added in the correct event and filtered it by sender = Gamma ALM